StandaloneMmPkg/Core/Dispatcher.c

```diff
@@ -655,7 +655,10 @@ FvIsBeingProcessed (
   DEBUG ((DEBUG_INFO, "FvIsBeingProcessed - 0x%08x\n", FwVolHeader));
 
   KnownFwVol = AllocatePool (sizeof (KNOWN_FWVOL));
-  ASSERT (KnownFwVol != NULL);
+  if (KnownFwVol == NULL) {
+    ASSERT (KnownFwVol != NULL);

```

Comment:
Since the error condition has already been evaluated, is it possible to change the assert statement to 'ASSERT (FALSE);', please?

Reply:
@samimujawar , sure no problem. I have updated this, can you please re-review?